src/sagemaker_xgboost_container/distributed_gpu/dask_data_utils.py

```diff
@@ -32,7 +32,7 @@ def _read_data(local_path: str, content_type: str) -> (DataFrame, Series):
 
     target_column = dataframe.columns[0]
     labels = dataframe[target_column]
-    features = dataframe[dataframe.columns.difference([target_column])]
+    features = dataframe.drop(target_column, axis=1)
```

Comment:
Do you know if this will copy if the features and be a potential memory-related concern considering how we expect these to typically be massive datasets?

Reply:
Hmm ya, looks like drop creates a copy.

Reply:
Thanks, now changed to just pick all but the first column name.